pkg/perf/networkhealth/networkhealth.go

```diff
@@ -0,0 +1,111 @@
+package networkhealth
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/threefoldtech/zos/pkg/environment"
+	"github.com/threefoldtech/zos/pkg/perf"
+)
+
+const defaultRequestTimeout = 5 * time.Second
+
+type NetworkHealthTask struct{}
+
+type ServiceStatus struct {
+	Url         string `json:"url"`
+	IsReachable bool   `json:"is_reachable"`
+}
+
+var _ perf.Task = (*NetworkHealthTask)(nil)
+
+func NewTask() *NetworkHealthTask {
+	return &NetworkHealthTask{}
+}
+
+func (t *NetworkHealthTask) ID() string {
+	return "network-health"
+}
+
+func (t *NetworkHealthTask) Description() string {
+	return "Network health check runs periodically to check the connection to most of grid services."
+}
+
+func (t *NetworkHealthTask) Cron() string {
+	return "0 */5 * * * *"
+}
+
+func (t *NetworkHealthTask) Jitter() uint32 {
+	return 0
```

Comment:
we need to add jitter here or it means ALL nodes in the world will try to test the services at exactly the same time, DDoS(ing) our own services!

Reply:
I think this should be the number of seconds (check docs) but if we gonna do a test every 15 min the jitter could be around 5 min or so